.github/workflows/publish.yml

```diff
@@ -0,0 +1,26 @@
+name: Cargo Publish
+
+on:
+  push:
+    tags:
+      - 'v**'
```

Comment:
According to https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet, we can be more restrictive and use something like 
```yaml
- "v[0-9]+.[0-9]+.[0-9]+"
- "v[0-9]+.[0-9]+.[0-9]+-[a-zA-Z0-9]+.[0-9]+"
```
or maybe just the first one.

Reply:
Done.  Went with the first.